assets/styles/flairs.less

```diff
@@ -97,3 +97,23 @@ a[href="/lucky"], a[href="/egg"], a[href="/eevee"], a[href="/togepi"], a[href="/
 .flair2-beautyribbon:before { background-position: -44px 0;}
 .flair2-smartribbon:before { background-position: -66px 0; margin-bottom: -2px; }
 .flair2-toughribbon:before { background-position: -88px 0; margin-bottom: -2px; }
+
+.flair-rowlet:after, .flair-litten:after, .flair-popplio:after {
```

Comment:
Keeping the extra flairs in the config makes it easier to turn off the setting (I realise I haven't added a setting to remove it from the frontend, but meh). A **bit** of config is better than 0 config.